packages/amplify-cli/src/cli.js

```diff
@@ -1,6 +1,6 @@
 const { build } = require('gluegun');
 const path = require('path');
-const globalPrefix = require('global-prefix');
+const globalPrefix = require('./lib/get-global-prefix');
```

Comment:
@mlabieniec  I can't seem to remember the reason, but why had you recommended against using the global-prefix package?